3rd_party/licenses/eigen-INFO.csv

```diff
@@ -1,2 +1,2 @@
 name,version,revision,url,license,copyright
-Eigen,3.2.5,bdd17ee3b1b3a166cd5ec36dcad4fc1f3faf774a,http://eigen.tuxfamily.org,MPL-2.0,
+Eigen,3.3.7,323c052,http://eigen.tuxfamily.org,MPL-2.0,
```

Comment:
Please could you use the full SHA1 for consistency with the other `INFO.csv` files?  It's 323c052e17310984a40dda9426e0435bed7178fa.

(Also, for the benefit of any subsequent reviewers, 323c052e17310984a40dda9426e0435bed7178fa refers to the commit in the primary Mercurial repo which lives at https://bitbucket.org/eigen/eigen, not the Git mirror used in `pull-eigen.sh`.)